test/runnable/iasm.d

```diff
@@ -4786,6 +4786,18 @@ void test5012()
 
 /****************************************************/
 
+import core.vararg;
+extern(C) void func16400(int, ...) {
+    asm {naked; ret;};
+}
+
+void test16400()
+{
+    assert(*(cast(ubyte*) &func16400) == 0xc3); // ret
```

Comment:
Please add calling the function, too.

Have you tried extern(D) and extern(C++) functions aswell? Maybe would be good to include test cases for these, too.